Gifski/MainWindowController.swift

```diff
@@ -301,6 +305,14 @@ final class MainWindowController: NSWindowController {
 		}
 	}
 
+	@IBAction private func quickLook(_ sender: Any) {
+		// Kind of a hack to get our conversion view to be the next responder to answer
```

Comment:
@sindresorhus this is the hack I was talking about with moving the logic. Interestingly enough, I had to clear the `nextResponder` as it got into some weird state afterwards.